Cargo.toml

```diff
@@ -12,15 +12,23 @@ edition = "2018"
 
 [features]
 default = []
-ssl = ["openssl"]
+ssl = ["ssl-openssl"]
+ssl-openssl = ["openssl"]
+ssl-rustls = ["rustls"]
 
 [dependencies]
 ascii = "1.0"
 chunked_transfer = "1"
-openssl = { version = "0.10", optional = true }
 url = "2"
-chrono = { version = "0.4", default-features = false, features=["clock"] }
 log = "0.4"
+time = { version = "0.3", features = [ "std", "formatting", "macros", "parsing" ] }
+
+#[cfg(feature = "ssl-openssl")]
```

Comment:
`#[cfg()]` directives don't do anything in `Cargo.toml` so these can be removed.